writeup/gz_hubble_data.tex

```diff
@@ -99,7 +99,9 @@
+We present the data release paper for the Galaxy Zoo: Hubble (GZH) project.\footnote{This publication has been made possible by the participation of more than 200,000 volunteers in the Galaxy Zoo project. Their contributions are individually acknowledged at \url{http://authors.galaxyzoo.org/authors.html}.} This is the third phase in a large effort to measure reliable, detailed morphologies of galaxies by using crowdsourced visual classifications of color composite images. Images in GZH are selected from various \textit{Hubble Space Telescope} Legacy programs using the Advanced Camera for Surveys (AEGIS, COSMOS, GEMS, GOODS-N, and GOODS-S), with filters selected to probe the rest-frame optical emission from galaxies out to $z\simeq1$. The galaxies selected for GZH classifications go down to magnitude limits of $m_{I814W}<23.5$ and have a median redshift of $\langle z\rangle=0.9\pm0.6$, with a tail extending out to $z\simeq4$. The GZH morphological data includes measurements of both bulge- and disk-dominated galaxies, details on spiral disk structure that relate to the Hubble type, and numerous measurements of clump identification and geometry. This paper also describes and employs a novel method for calibrating morphological bias by using artificially-redshifted galaxy images as a baseline. The GZH catalog includes both the raw and calibrated morphological vote fractions for $150,771$ galaxies, providing a comprehensive data set suitable for large-scale studies of galaxy evolution. 
```

Comment:
yeah, this doesn't help much, does it?

"image" --> "images" in second non-footnote sentence
rewrite of start of that same sentence because I don't know what "detailed scales" means